main.go

```diff
@@ -160,12 +160,20 @@ func SlightEchoHandler(writer http.ResponseWriter, request *http.Request) {
 	writer.Write(responsePayload)
 }
 
-func runWorker(c *cli.Context, n *server.Node) error {
+func runWorker(c *cli.Context, n *server.Node, g peer.ID) error { // TODO: Is there a reason all these have just c,n as arguments?
 	time.Sleep(10 * time.Second)
+	var f, release = n.Cluster.View().Lookup(c.Context) // TODO: Submit a query for ("gatewayID/lb/chan")?
```

Comment:
You already know the gateway's ID now.  So what you should do is `n.Vat.Connect(...)` to get the channel from the gateway.

Pro tip:  when connecting, don't worry about the gateway peer's address.  If you just pass in `&peer.AddrInfo{ID: g}`, wetware will use the DHT to figure out the rest.

Reply:
I am confused what I should put in as the 3rd argument of n.Vat.Connect()

Reply:
Like the first two seem straightforward, but I'm not sure how I get the capability to put into the 3rd slot.

Reply:
See [here](https://pkg.go.dev/github.com/wetware/casm/pkg#Capability).  Remember last time we met, how we dug into how capabilities were mapped onto libp2p streams?  

In your case, you'll want to use [`BasicCap`](https://pkg.go.dev/github.com/wetware/casm/pkg#BasicCap) to define which stream will service your channel.  If you grep for `BasicCap`, you should see some straightforward examples.

Reply:
Ok thank you!